data/exchange_versions.json

```diff
@@ -0,0 +1,322 @@
+{
+  "exchange_builds":
```

Comment:
Pretty printing this file would be good, the formatting is a bit off 👀 

```
ruby -r 'json' -e "puts JSON.pretty_generate(JSON.parse(File.read('data/exchange_versions.json')))"
```

Reply:
Thanks used this command and applied the resulting formatting 😄

Reply:
Fixed in https://github.com/rapid7/metasploit-framework/pull/16249/commits/c98666dc9284f1472ed801896dbd775947bb2d1b